tests/napari/test_worker_signals.py

```diff
@@ -0,0 +1,76 @@
+from pytestqt.qtbot import QtBot
+
+from cellfinder.napari.detect.detect_containers import (
+    ClassificationInputs,
+    DataInputs,
+    DetectionInputs,
+    MiscInputs,
+)
+from cellfinder.napari.detect.thread_worker import Worker
+from cellfinder.napari.sample_data import load_sample
+
+
+def run_worker_test(
+    qtbot: QtBot,
+    skip_detection: bool,
+    skip_classification: bool,
+    expected_labels: list,
+):
+    data = load_sample()
```

Comment:
```suggestion
    """Helper function to test worker signals for the progress bar.
    
    Asserts that all elements in `expected_labels` are emitted as signal messages.
    """
    data = load_sample()
```
Or similar docstring to help understand this function's purpose.